frontend/package.json

```diff
@@ -8,7 +8,12 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "vue": "2.5.17"
+    "axios": "0.18.0",
+    "vue": "2.5.17",
```

Comment:
I guess it is warning instead of error, and app working. I've tried to upgrade vue, but it broke the app. This caused lot of another incompatibilities. I can try play with it more, but I think we could make also separate issue for that task.

Reply:
Then you are right. Let's leave it how it is for now. I will just make an issue out of it in case anybody wants to tackle this one.